frontend/apps/reader/modules/readerfont.lua

```diff
@@ -404,6 +433,52 @@ These fonts will be used in this order:
 You can set a preferred fallback font with a long-press on a font name, and it will be used before these.
 If that font happens to be part of this list already, it will be used first.]]),
             table.concat(self.ui.document.fallback_fonts, "\n")),
+    })
+    table.insert(settings_table, {
+        text = _("Adjust fallback font sizes"),
+        checked_func = function()
+            return G_reader_settings:nilOrTrue("cre_adjusted_fallback_font_sizes")
+        end,
+        callback = function()
+            G_reader_settings:flipNilOrTrue("cre_adjusted_fallback_font_sizes")
+            self.ui.document:setAdjustedFallbackFontSizes(G_reader_settings:nilOrTrue("cre_adjusted_fallback_font_sizes"))
+            self.ui:handleEvent(Event:new("UpdatePos"))
+        end,
+        help_text = _([[
+Adjust the size of each fallback font so they all get the same x-height, and lowercase characters picked in them look similarly sized as those from the defaut font.
+This may help with greek words inside latin text (as latin fonts often do not have all the greek characters), but may make Chinese or Indic characters smaller when picked from fallback fonts.]]),
+        separator = true,
+    })
+
+    table.insert(settings_table, {
+        text_func = function()
+            local scale = G_reader_settings:readSetting("cre_monospace_scaling") or 100
+            return T(_("Monospace fonts scaling: %1 %"), scale)
+        end,
+        callback = function()
+            local SpinWidget = require("ui/widget/spinwidget")
+            UIManager:show(SpinWidget:new{
+                value = G_reader_settings:readSetting("cre_monospace_scaling") or 100,
+                value_min = 30,
+                value_step = 1,
+                value_hold_step = 5,
+                value_max = 150,
+                unit = "%",
+                title_text = _("Monospace font scaling"),
+                -- no info_text: we want this widget to stay small, so we can move it
+                -- around to see the effect of the scaling
+                keep_shown_on_apply = true,
+                callback = function(spin)
+                    local scale = spin.value
+                    G_reader_settings:saveSetting("cre_monospace_scaling", scale)
+                    self.ui.document:setMonospaceFontScaling(scale)
+                    self.ui:handleEvent(Event:new("UpdatePos"))
+                end
+            })
+        end,
+        help_text = _([[
+Monospace fonts may look big when inline with your main font if it has a small x-height.
+This allows scaling all monospace fonts by this percentage, so they can fit your preferred font height or have them be a bit smaller to distinguish it more easily.]]),
```

Comment:
Maybe something like this?
```suggestion
This setting allows scaling all monospace fonts by this percentage so they can fit your preferred font height, or you can make them a bit smaller to distinguish it more easily.]]),
```